doc/ChangeLog

```diff
@@ -1,4 +1,165 @@
+
+Purpose and description of changes
+----------------------------------
+
+ Notes copied here from the PR #1861:
+
+ A few substantial changes in the SNICAR module for the following updated snow
+ albedo calculation features:
+ - Updated ice optical properties from Flanner et al. (2021), with multiple types
+ for ice refractive indices.
+ - Updated aerosol optical properties from Flanner et al. (2021) with multiple
+ dust types & new BC and OC optics.
+ - Updated downward solar spectra from Flanner et al. (2021) for multiple
+ condition types.
+ - More accurate radiative transfer solver (adding-doubling) from Dang et al. (2019).
+ - Nonspherical snow grain scheme from He et al. (2017).
+ - BC-snow internal mixing scheme from He et al. (2017).
+ - Dust-snow internal mixing scheme from He et al. (2019).
+ - Hyperspectral (480-band, 10-nm spectral res) capability with all the above features.
+ - New namelist controls for aerosol in snow and additional snow albedo
+ diagnostic output variables.
+
+ Specific notes
+ - Code contributors: Cenlin He (NCAR/RAL) with advice from
+ Dave Lawrence (NCAR/CGD) and Mark Flanner (UMich).
+ - The manuscript to report this update is
+ Cenlin He, Mark Flanner, David M Lawrence, Yu Gu: New features and
+ enhancements in Community Land Model (CLM5) snow albedo modeling: description,
+ sensitivity, and evaluation. Authorea. June 08, 2023.
+ DOI:10.22541/essoar.168626390.01530324/v1
+ - These updates will change the snow and surface albedo results along with
+ other surface fluxes changes.
+ - There are a few new namelist options related to SNICAR scheme added to the
+ namelist control.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[X] clm5_1
+
+[X] clm5_0
+
+[X] ctsm5_0-nwp
+
+[X] clm4_5
+
+
+Bugs fixed or introduced
+------------------------
+CTSM issues fixed (include CTSM Issue #):
+
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+ New namelist variables:
+ snicar_numrad_snw = 5 or 480 wavelength bands, default 5
+ snicar_solarspec, default "mid_latitude_winter" among six available options
+ snicar_dust_optics, default "sahara" among three avail. options
+ snicar_snw_shape, default "hexagonal_plate" among fourn avail. options
+ snicar_use_aerosol, default .true.
+ snicar_snobc_intmix and snicar_snodst_intmix, default .false. means do not
+   activate bc-snow and dust-snow internal mixing
+
+ do_sno_oc, default .false., already appeared in previous code but in caps
+ snicar_aerforc_diag, default .false., existed before as use_snicar_frc
```

Comment:
Let's change this so that it has the use_ prefix.

And maybe if there isn't enough of a reason to change the name we go back to the old name?